CPE200Lab1/CPE200Lab1/Class1.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CPE200Lab1
+{
+    public class CalculatorEngine
+    {
+        public string calculate(string operate, string firstOperand, string secondOperand, int maxOutputSize = 8)
+        {
+            switch (operate)
+            {
+                case "+":
+                    return (Convert.ToDouble(firstOperand) + Convert.ToDouble(secondOperand)).ToString();
+                case "-":
+                    return (Convert.ToDouble(firstOperand) - Convert.ToDouble(secondOperand)).ToString();
+                case "X":
+                    return (Convert.ToDouble(firstOperand) * Convert.ToDouble(secondOperand)).ToString();
+                case "÷":
+                    // Not allow devide be zero
+                    if (secondOperand != "0")
```

Comment:
what will happen if this condition is false? user will get no response, right? which is not good, it should show Error.